apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -570,6 +577,20 @@ const EventTypePage = (props: EventTypeSetupProps) => {
                 }
               }
             }
+
+            // Prevent two payment apps to be enabled
+            let enabledPaymentApps = 0;
+            for (const appKey in metadata?.apps) {
+              const app = metadata?.apps[appKey as keyof typeof appDataSchemas];
+              if (app.price && app.enabled) {
+                enabledPaymentApps++;
+              }
+
+              if (enabledPaymentApps > 1) {
+                throw new Error(t("event_setup_multiple_payment_apps_error"));
+              }
+            }
```

Comment:
Why not use checkForMultiplePaymentApps in here as well? We already extracted the logic.